packages/browser/test/index.test.ts

```diff
@@ -1,119 +1,162 @@
-import { SentryEvent } from '@sentry/core';
+import { Context, SentryEvent } from '@sentry/core';
 import { expect } from 'chai';
 import * as RavenJS from 'raven-js';
 import { spy, stub } from 'sinon';
-import { BrowserBackend, BrowserFrontend, SentryClient } from '../src';
+import {
+  addBreadcrumb,
+  BrowserBackend,
+  BrowserFrontend,
+  captureEvent,
+  captureException,
+  captureMessage,
+  create,
+  popScope,
+  pushScope,
+  setExtraContext,
+  setTagsContext,
+  setUserContext,
+} from '../src';
 
 const dsn = 'https://53039209a22b4ec1bcc296a3c9fdecd6@sentry.io/4291';
 
 describe('SentryBrowser', () => {
-  beforeEach(async () => {
-    await SentryClient.create({ dsn });
+  before(() => {
+    create({ dsn });
```

Comment:
Just curious, why aren't the test cases isolated anymore? I saw the `s.restore()`, but just worried that due to jest's parallel execution, this could lead to interesting effects...

Reply:
We have one global install with `create` all other related tests push a seperate scope.
So this should be fine.